CHANGELOG.md

```diff
@@ -1,12 +1,13 @@
-# NEXT RELEASE
+# 5.12.5 Release notes
 
 ### Enhancements
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
```

Comment:
@finnschiermer @jedelbo 
Please notice and follow the hints in the template ;-) ^
How was the problem observed and what was the effect? And add a link to the issue/PR. That will detail what the problem was and how it was solved and doesn't have to be mentioned in the release note.